components/datetime/src/fieldset.rs

```diff
@@ -987,7 +987,7 @@ impl_zone_marker!(
     ///
     /// let fmt = FixedCalendarDateTimeFormatter::<Gregorian, _>::try_new(
     ///     &locale!("en").into(),
-    ///     Z::short(),
```

Comment:
This test is specifically documented as demonstrating the behavior when the display name is not available.